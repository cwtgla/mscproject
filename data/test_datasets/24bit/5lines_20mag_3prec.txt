1048575.1234
790034.3334
0.10238767
-55000.9715
-1048575.762859