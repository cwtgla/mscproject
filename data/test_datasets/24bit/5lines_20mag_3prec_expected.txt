127
255
249
96
112
147
0
0
1
134
182
193
255
255
255